app/code/Magento/AdvancedSearch/Test/Unit/Block/SearchDataTest.php

```diff
@@ -6,7 +6,7 @@
 
 namespace Magento\AdvancedSearch\Test\Unit\Block;
 
-use PHPUnit_Framework_MockObject_MockObject as MockObject;
+use PHPUnit\Framework\MockObject\MockObject as MockObject;
```

Comment:
Looks like we don't need `as` part here. Please provide an automated way to perform such change and let's enforce is with test.